cypress-tests/cypress/support/commands.js

```diff
@@ -1055,31 +1118,33 @@ Cypress.Commands.add(
     }).then((response) => {
       logRequestId(response.headers["x-request-id"]);
       expect(response.headers["content-type"]).to.include("application/json");
-      if (response.status === 200) {
-        expect(response.body).to.have.property("currency");
-        if (resData["payment_methods"].length > 0) {
-          function getPaymentMethodType(obj) {
-            return obj["payment_methods"][0]["payment_method_types"][0][
-              "card_networks"
-            ][0]["eligible_connectors"]
-              .slice()
-              .sort();
-          }
-          const config_payment_method_type = getPaymentMethodType(resData);
-          const response_payment_method_type = getPaymentMethodType(
-            response.body
-          );
-          for (let i = 0; i < response_payment_method_type.length; i++) {
-            expect(config_payment_method_type[i]).to.equal(
-              response_payment_method_type[i]
+      cy.wrap(response).then(() => {
```

Comment:
move one line up and:

```suggestion

      cy.wrap(response).then(() => {
```